test/hotspot/jtreg/compiler/arraycopy/TestObjectArrayClone.java

```diff
@@ -26,21 +26,304 @@
  * @bug 8155643
  * @summary Test Object.clone() intrinsic if ReduceInitialCardMarks is disabled.
  *
- * @run main/othervm -XX:+IgnoreUnrecognizedVMOptions -Xcomp -XX:-ReduceInitialCardMarks
- *                   -XX:CompileCommand=compileonly,compiler.arraycopy.TestObjectArrayClone::test
+ * @run main/othervm -XX:+IgnoreUnrecognizedVMOptions -XX:-ReduceInitialCardMarks
```

Comment:
Considering the complications `-XX:+StressReflectiveCode` introduces, does it make sense to add a configuration with the flag explicitly enabled?

Reply:
We have this test: compiler.arraycopy.TestEliminateArrayCopy (and there are some additional variants) that reproduces the problem, it uses the StressReflectiveCode flag.